src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -212,42 +231,41 @@ def insert():
         )
 
         # Drop NANs in column osm_id
-        busses.dropna(subset=["osm_id"], inplace=True)
+        buses.dropna(subset=["osm_id"], inplace=True)
 
         # Create columns for bus_id and geometry in the offshore df
-        offshore["bus_id"] = 0
+        offshore["bus_id"] = pd.NA
         offshore["geom"] = Point(0, 0)
 
         # Match bus_id
         for index, wind_park in offshore.iterrows():
-            if not busses[
-                (busses["osm_id"] == wind_park["osm_id"])
-                & (busses["base_kv"] == wind_park["Spannungsebene in kV"])
+            if not buses[
+                (buses["osm_id"] == wind_park["osm_id"])
+                & (buses["base_kv"] == wind_park["Spannungsebene in kV"])
             ].empty:
-                bus_ind = busses[
-                    busses["osm_id"] == wind_park["osm_id"]
-                ].index[0]
-                offshore.at[index, "bus_id"] = busses.at[bus_ind, "bus_id"]
+                bus_ind = buses[buses["osm_id"] == wind_park["osm_id"]].index[
+                    0
+                ]
+                offshore.at[index, "bus_id"] = buses.at[bus_ind, "bus_id"]
             else:
                 print(f'Wind offshore farm not found: {wind_park["osm_id"]}')
 
         offshore.dropna(subset=["bus_id"], inplace=True)
 
         # Overwrite geom for status2019 parks
-        if scenario == "status2019":
-            offshore["geom"] = offshore["Name ONEP/NEP"].map(map_ONEP_areas())
-            offshore["weather_cell_id"] = -1
-            offshore.drop(
-                ["Name ONEP/NEP", "Inbetriebnahme"], axis=1, inplace=True
-            )
-        else:
-            offshore["weather_cell_id"] = offshore[
-                "Netzverknuepfungspunkt"
-            ].map(w_id)
-            offshore["weather_cell_id"] = offshore["weather_cell_id"].apply(
-                int
+        if scenario in ["eGon2035", "eGon100RE"]:
+            offshore["Name ONEP/NEP"] = offshore["Netzverknuepfungspunkt"].map(
+                assign_ONEP_areas()
             )
 
+        offshore["geom"] = offshore["Name ONEP/NEP"].map(map_ONEP_areas())
+        offshore["weather_cell_id"] = pd.NA
```

Comment:
This is similar to the comment above: Is it fine to set this here?

Reply:
It could probably be the case for the "bus_id" column. I think this column accepts NA values without throwing an error.